Assets/Scripts/State/GridTile.cs

```diff
@@ -12,14 +12,17 @@ public enum WallType
     [System.Serializable]
     public class GridTile
     {
-        public WallType Type { get; }
+        public WallType Type;
+        public WallBehaviour Wall;
```

Comment:
akonahle budeme pridavat nejaku vec co bude miesto steny tak tam pojde ina ref, napr aj ten player spawn  alebo nejaky collectable vec

Reply:
walltype to bude empty